code/modules/clothing/head/jobs.dm

```diff
@@ -359,7 +355,7 @@
 
 /obj/item/clothing/head/beret/centcom_formal
 	name = "\improper CentCom Formal Beret"
-	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Central Command's most recent nano-fabric durability enhancements, this time, it's not the case."
+	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Central Command's nano-fabric durability enhancements, it's not the case."
```

Comment:
why are you even touching this?

Reply:
It made sense at the time when everything was having armor removed since the old text was "most recent", because saying "most recent" when no other beret had armor didn't really feel correct to me personally. I rewrote it a little bit, but given the other recommended changes I'll probably revert it.